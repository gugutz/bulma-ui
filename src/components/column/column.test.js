import React from 'react'
import renderer from 'react-test-renderer'
import Column from './column'

describe('<Column /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Column>
          <div>Some content</div>
        </Column>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isOneThird prop', () => {
    const tree = renderer
      .create(
        <Column isOneThird>
          <div>Some content</div>
        </Column>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isHalf prop', () => {
    const tree = renderer
      .create(
        <Column isHalf>
          <div>Some content</div>
        </Column>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
