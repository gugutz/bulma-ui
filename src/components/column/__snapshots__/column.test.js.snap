// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Column /> component should render correctly 1`] = `
<div
  className="column"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Column /> component should support isHalf prop 1`] = `
<div
  className="column is-half"
>
  <div>
    Some content
  </div>
</div>
`;

exports[`<Column /> component should support isOneThird prop 1`] = `
<div
  className="column is-one-third"
>
  <div>
    Some content
  </div>
</div>
`;
